static/js/tools/map/fetcher/map_point_stat.ts

```diff
@@ -0,0 +1,83 @@
+/**
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * Fetch the map point stat data.
+ */
+
+import axios from "axios";
+import _ from "lodash";
+import { Dispatch } from "react";
+
+import {
+  EntityObservationWrapper,
+  PointApiResponse,
+} from "../../../shared/stat_types";
+import { stringifyFn } from "../../../utils/axios";
+import { ChartDataType, ChartStoreAction } from "../chart_store";
+import { getDate } from "../util";
+
+export function fetchMapPointStat(
+  parentEntity: string,
+  childType: string,
+  statVar: string,
+  date: string,
+  dispatch: Dispatch<ChartStoreAction>
+): void {
+  const action: ChartStoreAction = {
+    type: ChartDataType.mapPointStat,
+    error: null,
+    context: {
+      placeInfo: {
+        enclosingPlace: {
+          dcid: parentEntity,
+          name: "",
+        },
+        enclosedPlaceType: childType,
+      },
+      statVar: {
+        dcid: statVar,
+        date,
+      },
+    },
+  };
+  axios
+    .get<PointApiResponse>("/api/observations/point/within", {
+      params: {
+        child_type: childType,
```

Comment:
sorry, this is not relevant to this PR, but we should make flask endpoint params be camelcase. I believe that was the pattern we were using before

Reply:
sg. I will make a change in follow up PR